tests/integration/api/test_azure_integration.py

```diff
@@ -0,0 +1,81 @@
+import os
+from datadog import api as dog
+from datadog import initialize
+
+API_KEY = os.environ.get("DD_TEST_CLIENT_API_KEY", "a" * 32)
```

Comment:
Since they are in all files, let's move them to a common place.

Reply:
Thanks!